wazuh/wazuh-kibana-app
# Deploy new agent section

**Added:**
Operating systems,
Versions,
Architectures,
Commands of Install and enroll the agent y
Commands to Start the agent.
The unit tests are being worked on.

This PR solves this [issue](https://github.com/wazuh/wazuh-kibana-app/issues/4204) and this other [issue](https://github.com/wazuh/wazuh-kibana-app/issues/4178). It is currently a PR draft since for it to close, the information to be displayed when the user clicks on an operating system that is deprecated is required.
This PR updates the content of the 'Deploy new agent' section and improves the view.

**Steps to test:**
*Go to the Agents tab
*Click on the 'Deploy new agent' button
*Check that all the exposed data are those that appear in the following [issue](https://github.com/wazuh/wazuh-packages/issues/1625)
